lib/nodejs/kml-handler.js

```diff
@@ -0,0 +1,43 @@
+// kml-handler.js
+//  This file handles the sending/receiving of KML files between applications
+
+
+var net = require( "net" ),
+  fs = require( "fs" ),
+  formidable = require( "formidable" );
+
+
+
+function sendKML ( req, res, next ) {
+  var form = new formidable.IncomingForm();
+  form.keepExtensions = true;
+  // parse the file and then send response
+  form.parse( req, function( err, fields, files ) {
+    if ( !err ) {
+      var client = new net.Socket();
```

Comment:
I don't know much about `net`, but I know that elsewhere we have been using `socket.io` as our websocket wrapper.  Do you know if `net` is better?  Or should we maybe stick with `socket.io` to be consistent?